brands/amenity/parking.json

```diff
@@ -9,7 +9,11 @@
       "name": "Diamond Parking"
     }
   },
-  "amenity/parking|Indigo": {
+  "amenity/parking|Indigo~(France)": {
```

Comment:
You don't want to add ~(France) unless there is more than one Indigo parking brand to add. This should just be Indigo with the country code of FR. The same goes for the no_match. That won't match since there isn't actually a brand with that entry.

Reply:
Fair, and maybe that entry for Indigo is a different Indigo, but if you go here: https://www.group-indigo.com/en/group/ and scroll to the `SHAREHOLDERS` section there is a map of worldwide locations where this Indigo Group has parking garages etc...but yeah maybe it's not the same Indigo.